sdk/identity/identity-cache-persistence/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "@azure/azure-sdk/ts-package-json-module": "off",
+    "@azure/azure-sdk/ts-modules-only-named": "off",
+    "@azure/azure-sdk/ts-package-json-engine-is-present": "off"
```

Comment:
?

Reply:
I got rid of the default export and restored ts-modules-only named. The other two make too many assumptions. The `engines` field is set to Node >= 12.0.0 for these extension packages, and the `module` field has to be set deeper into the source tree because of our use of shared code from identity. Keyvault has the same configuration.

Reply:
Can you file an issue to fix the linter?